crates/humanode-peer/src/chain_spec.rs

```diff
@@ -252,6 +252,35 @@ const EXISTENTIAL_DEPOSIT: Balance = 500;
 /// The initial pot accounts balance for testnet genesis.
 const INITIAL_POT_ACCOUNT_BALANCE: Balance = EXISTENTIAL_DEPOSIT + DEV_ACCOUNT_BALANCE;
 
+/// A helper function to calculate bridge pot balances.
+#[allow(clippy::type_complexity)]
+fn get_bridge_pot_balances(
+    basic_native_accounts: &[(AccountId, Balance)],
+    basic_evm_accounts: &[(EvmAccountId, Balance)],
+) -> (Balance, Balance) {
```

Comment:
If we can take two `impl IntoIter` here instead of slices we can remove `.collect`s and thus eliminate extra iterations

Reply:
https://github.com/humanode-network/humanode/pull/700/commits/14e6cd096a332224ee79fbe21365b8da22b57154